### ISSUE TITLE
Pika CDN status

### ISSUE THREAD

**Issue by hsablonniere** [pikapkg/cdn#27]
Hey @FredKSchott :wink:

I was doing some research and testing on skypack and Pika CDN and I'm a bit confused.
It seems like Pika CDN is still up but it seems like it's not just a different domain but has a different build/system under the hood.

I also noticed that the [Pika CDN website](https://www.pika.dev/cdn) does not refer to skypack or any deprecation notice.

Can you tell us more about:

1. What's the future of the Pika CDN project and brand?
2. What will happen to URLs on this domain https://cdn.skypack.dev?
3. What are the differences behind the hood between Pika CDN and skypack?

Thanks.

**Comment by FredKSchott** [comment]
Thanks for posting! You can see the draft of how we'll be updating the Pika CDN landing this week, to explain the move and our deprecation schedule: https://pika-www-3.fkschott.now.sh/cdn. Just putting some finishing touches on it before deploying.

tl;dr: We'll stop supporting new, first-time packages on the Pika CDN in August, and start redirecting existing packages to Skypack towards the end of this year. The API is a drop-in match, so we expect zero disruption via automatic redirect and will test all URLs in use today before moving.

**Comment by FredKSchott** [comment]
Published to https://www.pika.dev/cdn !
